frontend/package.json

```diff
@@ -1,46 +1,34 @@
 {
-  "name": "preview-servant-frontend",
-  "version": "0.9.0",
-  "license": "MIT",
-  "private": true,
-  "scripts": {
-    "serve": "vue-cli-service serve",
-    "build": "vue-cli-service build"
-  },
-  "dependencies": {
-    "@fortawesome/fontawesome-free": "^5.15.4",
-    "@fortawesome/fontawesome-svg-core": "^1.2.36",
-    "@fortawesome/free-solid-svg-icons": "^5.15.4",
-    "@fortawesome/vue-fontawesome": "^3.0.3",
-    "@vue/compat": "^3.2.47",
-    "bootstrap-material-design": "^4.1.3",
-    "core-js": "^3.20.2",
-    "current-script-polyfill": "^1.0.0",
-    "esprima": "^4.0.1",
-    "jquery": "^3.6.0",
-    "moment": "^2.29.1",
-    "parse-link-header": "^2.0.0",
-    "popper.js": "^1.16.1",
-    "swagger-ui": "^4.18.2",
-    "vue": "^3.2.47",
-    "vue-router": "^4.1.6",
-    "vue-virtual-scroller": "^2.0.0-beta.8",
-    "vuex": "^4.1.0"
-  },
-  "devDependencies": {
-    "@babel/polyfill": "^7.12.1",
-    "@babel/preset-env": "^7.21.4",
-    "@vue/cli-plugin-babel": "^5.0.8",
-    "@vue/cli-plugin-vuex": "^5.0.8",
-    "@vue/cli-service": "^5.0.8",
-    "@vue/compiler-sfc": "^3.2.47",
-    "sass": "1.62.0",
-    "sass-loader": "^13.2.2"
-  },
-  "browserslist": [
-    "ie >= 11",
-    "Firefox >= 60",
-    "Firefox ESR",
-    "Chrome >= 60"
-  ]
+   "name": "preview-servant-frontend",
+   "version": "0.9.0",
+   "license": "MIT",
+   "private": true,
+   "scripts": {
+      "serve": "vite",
+      "build": "vite build"
+   },
+   "dependencies": {
+      "@fortawesome/fontawesome-free": "^5.15.4",
+      "@fortawesome/fontawesome-svg-core": "^1.2.36",
+      "@fortawesome/free-solid-svg-icons": "^5.15.4",
+      "@fortawesome/vue-fontawesome": "^3.0.3",
+      "@vue/compat": "^3.4.27",
+      "bootstrap-material-design": "^4.1.3",
+      "core-js": "^3.27.1",
+      "jquery": "^3.7.1",
+      "moment": "^2.30.1",
+      "http-link-header": "^1.1.3",
+      "popper.js": "^1.16.1",
+      "swagger-ui": "^4.19.0",
```

Comment:
Core-js has a new version - https://www.npmjs.com/package/core-js/v/3.37.1
Swagger-ui as well has newer versions which have more downloads.

Reply:
core-js is update. Swagger not yet because there is a breaking change for PREvant that prevents the upgrade to 5.x. I want to do that another time.